src/server.h

```diff
@@ -581,6 +581,15 @@ typedef enum {
 #define PAUSE_ACTION_EVICT (1 << 3)
 #define PAUSE_ACTION_REPLICA (1 << 4) /* pause replica traffic */
 
+/* Sets log format */
+typedef enum { LOG_FORMAT_DEFAULT = 0, LOG_FORMAT_LOGFMT } log_format_type;
+
+/* Sets log timestamp format */
+/* Also update LOG_TIMESTAMP_FORMATS */
+#define LOG_TIMESTAMP_DEFAULT 0
```

Comment:
The date format "default" should have a more descriptive name (and one that doesn't imply that it will always be the default). Is this format used anywhere else, like in email headers?

It contains the name of the moth in English, as in "6 Oct 2024", so can we call it "English" or "British" or something? (Americans would put month first, right?)

Reply:
How about `LOG_TIMESTAMP_DMY`

Reply:
More important is what's visible to users, the config. Just "dmy"? Hm...

Reply:
The decision was to use "legacy" for the legacy timestamp format too.